frontend/package.json

```diff
@@ -4,20 +4,34 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
-    "react": "^16.5.2",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.4",
     "react-bootstrap-table-next": "^3.0.0",
     "react-bootstrap-table2-paginator": "^2.0.3",
-    "react-dom": "^16.5.2",
-    "react-fontawesome": "1.5.0",
-    "react-router-dom": "4.3.1",
-    "react-scripts": "2.0.5",
+    "react-dom": "^16.8.4",
+    "react-fontawesome": "~1.5.0",
+    "react-router-dom": "~4.3.1",
+    "react-scripts": "~2.1.8",
     "reactstrap": "^7.1.0",
-    "terser": "^3.14.1"
+    "terser": "^3.17.0"
```

Comment:
We may also be able to remove this. I added this to explicitly downgrade this react-scripts dependency to avoid a bug in 3.15. But now that bug should be fixed, and we can probably just toss this.